vignettes/contribution_model.Rmd

```diff
@@ -17,50 +20,143 @@ knitr::opts_chunk$set(
 library(admiraldev)
```

Comment:
ummm...do we need admiraldev in this vignette?

Reply:
no, good catch